### PR TITLE
substituted deleted LIT3940Prayer and LIT4962Basema by LIT2993RepCh274

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/1557

### CODE REVIEW

**Review by MassimoVilla — APPROVED** [review]

File: Casamari/MNC010.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </msItem>
                   <msItem xml:id="ms_i3">
                      <locus from="34r" to="35v"/>
-                     <title type="complete" ref="LIT4962Basema">‘In the name of God, the First One without yesterday’</title>
+                     <title type="complete" ref="LIT2993RepCh274">‘In the name of God, the First One without yesterday’</title>
```
Comment (SusanneHummel) [inline Casamari/MNC010.xml:59]: I am currently working on this Ms, and changed the whole ms content, 
Is it needed to change it here? 
I am afraid of competing changes.

Comment (thea-m) [reply Casamari/MNC010.xml:59]: There might be a conflict, which will be very easy to solve, because this change is minimal

Comment (SusanneHummel) [reply Casamari/MNC010.xml:59]: So, why not leaving out this manuscript?

Comment (thea-m) [reply Casamari/MNC010.xml:59]: If your manuscript will be ready very soon, I can revert the change, but I am rather reluctant out of principle to knowingly use deleted IDs

**Comment by SusanneHummel** [comment]
The Ms description is almost ready. I just need to create the new work record for the hymn to God we discussed in the other issue, after that I can create the PR of the full description

**Comment by thea-m** [comment]
I have reverted it now

**Review by SusanneHummel — APPROVED** [review]

**Pull request merged**